src/common/api-interface.ts

```diff
@@ -36,6 +36,7 @@ export enum apiCmds {
     getConfigUrl = 'get-config-url',
     registerRestartFloater = 'register-restart-floater',
     setCloudConfig = 'set-cloud-config',
+    getCPUUsage = 'get-cpu-usage',
```

Comment:
name should perhaps be getCpuUsage, instead of getCPUUsage (more conventional to not use upper case for abbreviations in camel case)

Reply:
Will keep because of https://www.electronjs.org/docs/api/structures/cpu-usage